README.md

```diff
@@ -227,6 +227,7 @@ Dependabot and Dependabot Core started life as [Bump][bump] and
 GoCardless in helping make Dependabot possible - if you need to collect
 recurring payments from Europe, check them out.
 
+
```

Comment:
Intentional `noop` added to highlight the build failure before and after pinning to a specific version of faraday. I'll see if I can work upstream to help identify the root cause.